### PR TITLE
Removed load-components and unused storybook utils

### PR DESCRIPTION
**Pull request by tsantef**
## PR Checklist

Storybook can be viewed [here](https://docspot.adnxs.net/projects/lucid/load-components-cleanup/?path=/docs/documentation-intro--page)

- Manually tested across supported browsers

  - [x] Chrome
  - [ ] Firefox
  - [ ] Safari

- [ ] Unit tests written (`common` at minimum)
- [x] PR has one of the `semver-` labels
- [x] Two core team engineer approvals
- [ ] One core team UX approval

Screen Shot 2021-11-09 at 9 51 07 AM

### CODE REVIEW

**Review by lennoxdavidjr — APPROVED** [review]

**Review by yasskin — APPROVED** [review]

**Comment by yasskin** [comment]
I added a docspot Storybook that shows the "Private" components.
https://docspot.adnxs.net/projects/lucid/load-components-cleanup/?path=/docs/documentation-intro--page

File: src/components/Table/Table.stories.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import createClass from 'create-react-class';
 import { Button, Checkbox, SuccessIcon, Table } from './../../index';
 
 export default {
-	title: 'Table',
```
Comment (yasskin) [inline src/components/Table/Table.stories.tsx:6]: I think Table belongs in Table/Table.

**Review by yasskin — APPROVED** [review]

**Comment by yasskin** [comment]
The `ExampleComponent` is failing for me in local tests.
The Storybook story for it points to a peek.description but none exists.
```
	parameters: {
		docs: {
			description: {
				component: (ExampleComponent as any).peek.description,
			},
		},
	},
```

File: src/components/Table/Table.stories.tsx (hunk 1 above)
Comment (yasskin) [reply src/components/Table/Table.stories.tsx:6]: 👍

**Pull request merged**